main.js

```diff
@@ -0,0 +1,148 @@
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+function catalogItem(item) {
+  var $items = document.createElement('div')
+  $items.classList.add('itemsDiv')
+  var $name = document.createElement('h4')
+  $name.classList.add('card-header')
+  $name.textContent = item.name
+  var $brand = document.createElement('span')
+  $brand.classList.add('card-subtitle')
+  $brand.textContent = item.brand
+  var $price = document.createElement('span')
+  $price.classList.add('price')
+  $price.textContent = '$' + item.price
+  var $img = document.createElement('img')
+  $img.classList.add('card-img')
+  $img.setAttribute('src', item.imageUrl)
+  $items.appendChild($name)
+  $items.appendChild($img)
+  $items.appendChild($brand)
+  $items.appendChild($price)
+  return $items
+}
+
+function allCatalogItems(items) {
+  var $container = document.createElement('div')
+  $container.classList.add('container')
+  var $row = document.createElement('div')
+  $row.classList.add('row')
+  var $heading = document.createElement('h3')
+  $heading.classList.add('heading')
+  $heading.textContent = 'Jamazon'
+  $container.appendChild($heading)
+  $container.appendChild($row)
+  for (var i = 0; i < items.length; i++) {
+    var $col = document.createElement('div')
+    $col.classList.add('col-md-4')
+    var $cards = catalogItem(items[i])
+    $col.appendChild($cards)
+    $row.appendChild($col)
+  }
+  return $container
+}
+
+allCatalogItems(app.catalog.items)
+
+function renderApp(state) {
+  var $view = document.querySelector('[data-view=catalog]')
+  if (state.view === 'catalog') {
+    $view.innerHTML = ''
+    $view.appendChild(allCatalogItems(state.catalog.items))
+  }
+  if (state.view === 'items') {
```

Comment:
Does your program ever change the app state to view "items"? If not, you can remove this `if` statement. Be careful not to add code that doesn't do anything.